hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/common/TestBlockDeletingService.java

```diff
@@ -195,8 +198,13 @@ public void testBlockDeletion() throws Exception {
     ContainerSet containerSet = new ContainerSet();
     createToDeleteBlocks(containerSet, conf, 1, 3, 1);
 
+    OzoneContainer ozoneContainer = Mockito.mock(OzoneContainer.class);
+    Mockito.when(ozoneContainer.getContainerSet())
+        .thenReturn(containerSet);
+    Mockito.when(ozoneContainer.getWriteChannel())
+        .thenReturn(null);
```

Comment:
Since it is used in all the test classes, we can add the logic for mocking ozoneContainer in a TestUtil class.

Reply:
Addressed in the latest patch.